packages/web-components/src/components/va-maintenance-banner/va-maintenance-banner.tsx

```diff
@@ -0,0 +1,150 @@
+import { Component, Event, EventEmitter, Host, Prop, h } from '@stencil/core';
+import classNames from 'classnames';
+import {
+  formatDate,
+  isDateAfter,
+  isDateBefore,
+  isDateSameDay,
+} from '../../utils/date-utils';
+
+/**
+ * @componentName Maintenance Banner
+ * @maturityCategory use
+ * @maturityLevel deployed
```

Comment:
I am not certain of this maturity level. One of the criteria I see for [USE:DEPLOYED](https://design.va.gov/about/maturity-scale#use-deployed) says:
- Is deployed in production for more than 3 months

I wonder if this should be [USE WITH CAUTION: AVAILABLE](https://design.va.gov/about/maturity-scale#use-with-caution-available) for now?

```
 * @maturityCategory caution
 * @maturityLevel proposed
```

Reply:
Updated

Reply:
@powellkerry eeep, I think I pasted the wrong suggestion. The "available" reference should be this:

```
@maturityLevel available
```

Reply:
Updated